enterprise/admin/views.py

```diff
@@ -273,6 +292,17 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
                     course_id=course_id,
                     course_mode=mode
                 )
+                if notify:
+                    send_notification_message(
+                        request,
+                        pending_user,
+                        course_details['name'],
+                        course_url,
+                        _('course'),
```

Comment:
Along the lines of readability, this private/internal function is orchestrating a fair amount of work and it might be better to break it up into additional private/internal functions.  I understand that it's possible to go too far with that approach, but it would help to separate things a bit more, imho.